spec/support/common_helpers.rb

```diff
@@ -150,10 +150,10 @@ def transition(machine, state_event, user)
   # Closes a modal dialog and waits until it's really closed
   #
   def close_modal_dialog
-    within(".modal-diaglog") do
+    within(".modal") do
       click_link "Back"
     end
-    page.should_not have_css(".modal-backdrop", :wait => 30)
+    page.should_not have_css(".modal-backdrop", :wait => 15)
```

Comment:
Sorry I didn't get you, the commit 500843e6cabc26e3f604cf21fe649704a130ccfe contains only some of the corrections for the commit 372efc494ffc0a6aba55d75fad1bd578277775db. So should I squash them?